conconartist/lyric-lava
# Feature/add form testing

#### Purpose: 
> Check all that apply.
### What does this PR do? 
> Clearly and concisely describe changes made. 

Tests added in Cypress folder under integration to test functionality of the main page features.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

In the cypress folder of the project root directory, open the integration folder and pull up 'App.spec.js' in your script editor to view code blocks for testing.

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Clone this repository and run `npm install` and `npx cypress open` to run tests in cypress.  Under the integration folder, you should see App.spec.js. Click on the folder and run the tests.

#### Checklist:
